### PR TITLE
Feature: Password reset 

### PR DESCRIPTION
**Pull request by MartynaKolakowska**
https://github.com/codeandpepper/janush/issues/39

### CODE REVIEW

File: schematics/web/authentication/files/src/routing/routes/ForgotPassword/ForgotPassword.test.tsx.template [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { MemoryRouter } from "react-router-dom";
+import { ThemeProvider } from "@features/ThemeProvider/ThemeProvider";
+import { act, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ForgotPassword from "./ForgotPassword";
```
Comment (przemyslaw-rzasa-cp) [inline schematics/web/authentication/files/src/routing/routes/ForgotPassword/ForgotPassword.test.tsx.template:5]: https://github.com/codeandpepper/janush/blob/main/CONVENTIONS.md#modules-import

File: schematics/web/authentication/files/src/routing/routes/ForgotPassword/ForgotPassword.test.tsx.template [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  <ThemeProvider>
+    <MemoryRouter>
+      <ForgotPassword />
+    </MemoryRouter>
+  </ThemeProvider>
+);
+
+describe("ForgotPassword", () => {
+  it("should show forgot password form view", () => {
+    render(<MockForgotPassword />);
```
Comment (przemyslaw-rzasa-cp) [inline schematics/web/authentication/files/src/routing/routes/ForgotPassword/ForgotPassword.test.tsx.template:18]: for readability I'd. suggest to leave an empty space after `render`

File: schematics/web/authentication/utils/index.ts [hunk 3]
```diff
@@ -463,7 +467,9 @@ export const changeRoutes = (projectName: string): TreeFunction => {
         attr.getStart() - 1,
         "<Route path={Paths.SIGN_IN_PATH} component={SignIn} />\n" +
           "<Route path={Paths.SIGN_UP_PATH} component={SignUp} />\n" +
-          "<Route path={Paths.VERIFY_EMAIL_PATH} component={VerifyEmail} />\n",
+          "<Route path={Paths.VERIFY_EMAIL_PATH} component={VerifyEmail} />\n" +
+          " <Route path={Paths.FORGOT_PASSWORD_PATH} component={ForgotPassword} />\n" +
```
Comment (przemyslaw-rzasa-cp) [inline schematics/web/authentication/utils/index.ts:471]: Is this empty space left here intentionally?

**Review by przemyslaw-rzasa-cp — COMMENTED** [review]
Some checks are failing. Please take a look on it

**Review by hadrysm — APPROVED** [review]
When this PR (#232) goes main, it should solve the problem of your test pipeline. Merge `main` into your branch.

**Pull request merged**